### PR TITLE
Upgrade IntelliJ Idea CE with bundled JDK to 14.1

### PR DESCRIPTION
**Pull request by sebroeder**
- Change download URL to https

### CODE REVIEW

File: Casks/intellij-idea-ce-bundled-jdk.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  app "IntelliJ IDEA #{version} CE.app"
-  
+  app "IntelliJ IDEA 14 CE.app"
+
   zap :delete => [
-                  "~/Library/Application Support/IdeaIC#{version}",
-                  "~/Library/Preferences/IdeaIC#{version}",
-                  "~/Library/Caches/IdeaIC#{version}",
-                  "~/Library/Logs/IdeaIC#{version}",
+                  "~/Library/Application Support/IdeaIC14",
```
Comment (vitorgalvao) [inline Casks/intellij-idea-ce-bundled-jdk.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

Comment (vitorgalvao) [inline Casks/intellij-idea-ce-bundled-jdk.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes. Better yet, these and the one in app should be changed to `#{version.to_i}`.

Comment (sebroeder) [inline Casks/intellij-idea-ce-bundled-jdk.rb]: > Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes. Better yet, these and the one in app should be changed to #{version.to_i}.

Done. Thanks for the hints.

**Comment by sebroeder** [comment]
@vitorgalvao I am having second thought whether `#{version.to_i}` is actually better here than hard-coding the number `14`. I would like to keep the many intellij-based formulas as similar as possible and `#{version.to_i}` would not work for [intellij-idea-ce-eap.rb](https://github.com/caskroom/homebrew-versions/blob/0898d7a0b0b4724a1b3ab1af18d14134bf94a0a5/Casks/intellij-idea-ce-eap.rb).

The "official" homebrew-cask formula [intellij-idea.rb](https://github.com/caskroom/homebrew-cask/blob/0bf58228c27979189d8447d03be133729eabf14b/Casks/intellij-idea.rb) has the version hard-coded, too.

And in the RubyMine formula [rubymine.rb](https://github.com/caskroom/homebrew-cask/blob/0bf58228c27979189d8447d03be133729eabf14b/Casks/rubymine.rb) you can see that a) Jetbrains naming scheme is not very predictable (they use `70` instead of `7` for RubyMine) and b) errors are harder to find once the code gets to clever (the regex is wrong in the RubyMine formula because it evaluates to `704` instead of `70`.

So I am on the fence to reverting back to the hard-coded version number (and single quotes). What do you think? /cc @radeksimko

**Comment by radeksimko** [comment]
> Jetbrains naming scheme is not very predictable

yeah, that sucks... +1 for hardcoding it then.

Generally speaking, if the vendor doesn't care about making these package versions/filenames predictable enough, we should not be spending time/effort by trying to work around that IMO.

**Comment by vitorgalvao** [comment]
Agreed. No quarrels at all about hardcoding.

**Comment by sebroeder** [comment]
Ready to be merged from my side.

**Pull request merged**